src/App.js

```diff
@@ -81,7 +81,7 @@ class App extends Component {
     !this.state.knackUserToken && navigate("/login");
 
     return (
-      <div className="container">
```

Comment:
I think we are going to need this container class for the other pages inside login for out Bootstrap layouts to work properly.